pkg/artifactory/resource/repository/remote/resource_artifactory_remote_repository_test.go

```diff
@@ -696,18 +696,17 @@ func mkNewRemoteTestCase(repoType string, t *testing.T, extraFields map[string]i
 		"retrieval_cache_period_seconds": 70,
 		// this doesn't get returned on a GET
 		//"failed_retrieval_cache_period_secs": 140,
-		"missed_cache_period_seconds":             2500,
-		"unused_artifacts_cleanup_period_enabled": true,
```

Comment:
Same here, is this field not needed any more?

Reply:
Yes, not needed.